allennlp/models/ensemble/bidaf_ensemble.py

```diff
@@ -0,0 +1,155 @@
+from typing import Dict, List, Any, Tuple
```

Comment:
I'm trying to think of what the right way is to structure this code.  I don't think it makes much sense to have an "ensembles" submodule, because we've already started organizing models around task, and this adds an orthogonal organization dimension.  Maybe put the main `Ensemble` class in `allennlp/models/ensemble.py`, and just add concrete ensembles next to the model they're ensembling (`models/reading_comprehension/bidaf_ensemble.py`)?  Then there's no need for an `allennlp/models/ensemble/` directory.

Reply:
Works for me.